Training/Training/file.c

```diff
@@ -3,39 +3,29 @@
 // Copyright (c) Gokul TS @Metamation India.
 // ------------------------------------------------------------------
 // file.c
-// C Program to implement Bubble sort and binary search algorithm with test cases
+// C Program to check test cases
 // ------------------------------------------------------------------------------------------------
+#include "SortAndSearch.h"
 #include <stdio.h>
 #include <stdint.h>
 #include <string.h>
 #include <ctype.h>
 #include <stdlib.h>
 
-#define MAX_DIGIT 13
-#define TEST_SIZE 7
-#define ANSI_RESET_ALL "\x1b[0m"
-#define ANSI_COLOR_RED "\x1b[31m"
-#define ANSI_COLOR_GREEN "\x1b[32m"
-#define ANSI_COLOR_YELLOW "\x1b[33m"
-
-/// <summary>Changes the input array into a sorted array</summary>
-void BubbleSort (int arr[], int size);
-/// <summary>Returns the index of key element in an array</summary>
-int BinarySearch (int sortArr[], int key, int size);
-/// <summary>Gets integer input from user</summary>
-int GetUserInput (char* prompt);
+/// <summary>Gets integer input from user with specified prompt</summary>
+int GetUserInputWithPrompt (char* prompt);
```

Comment:
Here only the user input is obtained from the user and prompt is provided in the program. But the current name is confusing that it gets both input and prompt from user.